tests/components/logbook/test_websocket_api.py

```diff
@@ -1181,6 +1181,10 @@ async def test_subscribe_unsubscribe_logbook_stream(
     await async_wait_recording_done(hass)
     websocket_client = await hass_ws_client()
     init_listeners = hass.bus.async_listeners()
+    init_listeners = {
+        **init_listeners,
+        EVENT_HOMEASSISTANT_START: init_listeners[EVENT_HOMEASSISTANT_START] - 1,
+    }
```

Comment:
This works, but it doesn't really explain why we do this. And adding a comment would work fine, but I can see a future where we add more like these kind of issues, and then it will become annoying to keep track of why we do the things we do